plugins/defaultplugins/l2plugin/fib_config.go

```diff
@@ -61,9 +61,9 @@ func (plugin *FIBConfigurator) Init() (err error) {
 	plugin.Log.Debug("Initializing L2 Bridge domains")
 
 	// Init local mapping
-	plugin.addCacheIndexes = bdidx.NewFIBIndex(nametoidx.NewNameToIdx(logrus.DefaultLogger(), "l2plugin", ""+
+	plugin.addCacheIndexes = l2idx.NewFIBIndex(nametoidx.NewNameToIdx(logrus.DefaultLogger(), "l2plugin", ""+
```

Comment:
Part with `"" + "..."` doesn't make sense.

Reply:
well, true but this is some automatic import-rename change so I missed it